.travis.yml

```diff
@@ -1,7 +1,7 @@
 sudo : false
 language: node_js
 node_js:
-  - "4.2"
+  - "6.9.1"
```

Comment:
It dictates which version of node Travis will use to execute the linting engine.  It has nothing to do with our JS or our build environment.